### DISCUSSION TITLE
Transfers

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#4072] [Q&A]
Hello,

I'm testing the transactiontyp "transfer"

If I click on advanced
left amount is the minus amount on the "from-account"
right amount is the plus amount on the "to-account"
Hope that I'm right with this.

Now for the test.

I have created a loan account.
There is a mounthly payment of 100 from the bank account.
This means:
Bank account -100 
=> Interest expense 10 
=> Transfer loan account 90 (repayment) 

What is the easiest way to post this?

**Comment by vomikan** [comment]
If I were you. I would first transfer money from account A to account B 1:1. The second transaction is two categories at once. The interest of the bank and the actual product.
In this case, detailed statistics will be available in reports.

**Reply by TomAff** [reply]
It's just a repayment. So I just need one expense account (category)
I think the best solution is to transfer 100 from bank account to loan account. => Bank balance is correct then
then a posting on the loan account => cash out => category interest. => Loan Balance is correct / Expense of 10 is correct in the P&L.

I just was wondering for what I can use the advance checkbox. In which cases do I have different amounts on a transfer?

**Reply by vomikan** [reply]
This is created for accounts in different currencies.
